source/agora/consensus/EnrollmentManager.d

```diff
@@ -193,28 +193,42 @@ public class EnrollmentManager
 
         Add a validator to the validator set or update the enrolled height.
 
+        This also gets and stores the information about this node's enrollment
+        information which could be lost in abnormal situations.
```

Comment:
Side note: What do you consider "abnormal situation" ? For me a node restarting is not abnormal. Nor is a node taking over another one.
What could be considered abnormal would be a catastrophic failure such as "disk unplugged". But "abnormal" makes me think it things that shouldn't happen, when they can and will happen, and we need to cater for it. Even if we equate "abnormal" to "catastrophic events", I think it is much better to describe what event, or what effect they would have (e.g. for "disk unplugged": inability to read / write to disk) than just calling it "catastrophic event".

But it's just a side note for future reference. I don't think it's worth fixing right now.